ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -1109,32 +1109,32 @@ describe('RuntimeConfigurationPanel', () => {
     expect(costEstimator(wrapper).exists()).toBeTruthy();
 
     // Default GCE machine, n1-standard-4, makes the running cost 20 cents an hour and the storage cost less than 1 cent an hour.
-    expect(runningCost(wrapper).text()).toEqual('$0.20/hour');
-    expect(storageCost(wrapper).text()).toEqual('< $0.01/hour');
+    expect(runningCost(wrapper).text()).toEqual('$0.20per hour');
```

Comment:
What do you think about adding a space before `per hour` in the component?  That would allow you to remove its marginLeft, and also its `text()` would look more correct.

(this is also fine)